libraries/3rdparty/mbedtls/include/mbedtls/config.h

```diff
@@ -1075,7 +1075,7 @@
  *
  * Enable functions that use the filesystem.
  */
-//#define MBEDTLS_FS_IO
+#define MBEDTLS_FS_IO
```

Comment:
We would also want to avoid changing this global config file anyway. Our approach with that is admittedly limiting, but I suspect that this demo could find a way to override the settings it needs.

Reply:
Yes, we don't need that define anymore -- just commented it back out. 

@Dan - I needed to uncomment some of the defines in the config file for CSR-related mbedTLS calls. I'm not sure if there is an effective workaround, but if so I'll look into implementing that.